pkgs/applications/editors/vim/plugins/vim-utils.nix

```diff
@@ -415,7 +415,10 @@ rec {
 
   toVimPlugin = drv:
     drv.overrideAttrs(oldAttrs: {
-      name = "vimplugin-${oldAttrs.name}";
+      name =
+        if lib.strings.hasPrefix "vimplugin" oldAttrs.name
+        then oldAttrs.name
+        else "vimplugin-${oldAttrs.name}";
```

Comment:
I am not sure about that. It's not a huge problem if the prefix is added several time IMO, aka we can catch it like you did.
And if we really wanted to make the name idempotent, it would be best to check if it's a vimplugin already via passthru.vimPlugin.

Reply:
I caught it only because I use [nix-output-monitor](https://github.com/maralorn/nix-output-monitor?tab=readme-ov-file#nix-output-monitor), otherwise I would never notice it.

Anyway, dropped https://github.com/NixOS/nixpkgs/pull/307167/commits/302fe727cf46ff349ee9328414741f187a5d3769

Reply:
ty. Sry forgot to merge :)